dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpCompatible-1.0.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpCompatible-1.0
```

Comment:
suggest to use mpComptible-pre4

Reply:
I think it has to be a number at the end if we want anything to be able to tolerate both 4.0 and pre-4.0

Reply:
exactly, can't do the tolerates if it doesn't follow the normal format

Reply:
Might still be worth a comment in this file to say what depending on `mpCompatible-1.0` means as it may confuse people.

Reply:
Hopefully this is more obvious now that I have added 2.0 and 3.0

Reply:
I think it's more confusing to have mpCompatible-2.0 and 3.0. It's not like we have an mpCompatible feature for every MP version, and the mpCompatible-2.0 and 3.0 features don't actually do anything (everything that uses any of them tolerates the others too).

I'd suggest putting a big comment on mpCompatible-1.0 saying that all MP features pre-3.3 should tolerate it. MP features for 4.0 or later should tolerate the relevant mpCompatible feature.

Reply:
we have decided to use mpCompatible-0.0